clients/linux/src/app.rs

```diff
@@ -603,7 +603,18 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
+                        Ok(renamed_meta) => {
+                            lb.gui.win.set_title(&renamed_meta.name);
+                            lb.gui.account.set_search_field_text(&lb.core.full_path_for(&renamed_meta));
+                            lb.messenger.send(Msg::RefreshSyncStatus);
+                        }
+                        Err(err) => {
+                            d.close();
+                            lb.messenger.send_err("renaming file", err);
```

Comment:
This wouldn't be an error `"renaming file"`, it would be an error like `"getting renamed file"`.